Elbow Saver/Resources/Strings.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Strings.swift
+//  Strings
+//
+//  Created by Daniel Luo on 8/27/21.
+//
+
+import Foundation
+
+enum Strings {
+    enum Timer {
+        static let stopped = NSLocalizedString(
+            "timer-stopped",
+            tableName: nil,
+            bundle: Bundle.main,
+            value: "Stopped",
+            comment: "Timer is stopped, not currently running."
+        )
+        static let reset = NSLocalizedString(
+            "timer-reset",
+            tableName: nil,
+            bundle: Bundle.main,
+            value: "RESET",
+            comment: "Cue for user to reset position between reps of exercise"
+        )
+        static func restWith(timeRemaining: Int) -> String {
+            let formatString = NSLocalizedString(
+                "timer-rest",
+                tableName: nil,
+                bundle: Bundle.main,
+                value: "REST: %ld",
+                comment: "Cues user to rest between sets, with time remaining in seconds."
+            )
+            return String.localizedStringWithFormat(formatString, timeRemaining)
+        }
+        static func repCounter(currentRep: Int, totalRepsInSet: Int) -> String {
+            let formatString = NSLocalizedString(
+                "timer-rep-counter",
+                tableName: nil,
+                bundle: Bundle.main,
+                value: "%ld/%ld Reps",
+                comment: "Displays user's progress of reps in an exercise set"
+            )
+            return String.localizedStringWithFormat(formatString, currentRep, totalRepsInSet)
+        }
+        static func setCounter(currentSet: Int, totalSetsInSession: Int) -> String {
```

Comment:
Lol, a "gotcha" for your project only. You may want to rename this `setsCounter` because it looks like `set` as in "set property"

Reply:
Only an issue when it's at the beginning of a string, since it looks like a verb. at end of string it's clear enough that it's a noun (e.g. `currentSet`)

Reply:
Dang it actually was `setsCounter` when I first wrote it lol derp